cookie.css

```diff
@@ -0,0 +1,37 @@
+.cookie-banner {
+    background: #05303F;
+    border: 5px solid #F5592C;
+    box-sizing: border-box;
+    border-radius: 10px;
+    width: 100vw;
+    font-family: 'Roboto';
+    font-style: normal;
+}
+.cookie-banner p {
+    font-weight: normal;
+    font-size: 15px;
+    line-height: 18px;
+    text-align: left;
+    padding: 0 10px;
+}
+.cookie-setting-bttn, .cookie-accept-bttn {
```

Comment:
Not an issue, just a note, that "bttn" is not a commonly used abbreviation, but rather `btn` :)